### DISCUSSION TITLE
Installing starter kit on an existing site

### DISCUSSION THREAD

**Discussion by joybcode** [duncanmcclean/simple-commerce#1006] [Help] [answered]
Hi,

I'm new to statamic and have started a project locally. Is it possible to intall the starter kit (with page templates etc) into an existing site?

**Comment by duncanmcclean** [comment, marked as answer]
Yes, you can install starter kits into existing sites with the following command:

```php
php please starter-kit:install duncanmcclean/sc-starter-kit
```

**Comment by joybcode** [comment]
Hi Duncan

Thank you for getting back to me so quickly.

I purchased the simple commerce addon from the statamic site. I am working on a site on my local server which has already got pages/templates and blueprints.


However, I installed simple commerce using your suggestion: php please starter-kit:install duncanmcclean/sc-starter-kit
This replaced a lot of my existing content and blueprints.

Is there an easier way to just copy specific pages/templates for products/cart/checkout pages?

I look forward to hearing from you.

Kind Regards

Joy Binning | Head of Marketing

[Dentalserve-Logo-for-Emails]

Tel: 01474 614 017<tel:014743614017>

**Reply by duncanmcclean** [reply]
Sure, just [download the zip](https://github.com/duncanmcclean/sc-starter-kit) from the starter kit repository (`Code -> Local -> Download ZIP`) and copy what you need/want.

**Comment by joybcode** [comment]
Hi Duncan,

Thanks once again.

Do you have any video tutorials on how to setup the shop/product/cart/checkout pages?

Kind Regards

Joy Binning | Head of Marketing

[Dentalserve-Logo-for-Emails]

Tel: 01474 614 017<tel:014743614017>

**Reply by duncanmcclean** [reply]
I don't have any videos, sorry!
